src/editor/editor/style.scss

```diff
@@ -2,3 +2,8 @@
 .editor-post-preview__dropdown {
 	display: none;
 }
+
+// TODO hide title after it's not required in MC initial call
```

Comment:
The "TODO" bit is not needed here ;)

Reply:
Right, done in https://github.com/Automattic/newspack-newsletters/pull/40/commits/fffbcd25eca2b7637f162566eee1725ba90de19a